.github/workflows/test.yml

```diff
@@ -23,7 +23,11 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install flake8
+        python -m pip install flake8 black==22.3.0
+    # Ignore E203, W503 and W504 which is against PEP 8 style
     - name: flake8
       run: |
-        flake8 ovn-tester
+        flake8 --ignore=E203,W504,W503 .
```

Comment:
This now runs flake8 on more files which is a good thing.  But we should also fix the warnings it raises in those files, e.g.:
```
./ovn-fake-multinode-utils/process-monitor.py:51:9: E722 do not use bare 'except'
./utils/process-stats.py:2:1: F401 'numpy' imported but unused
./utils/process-stats.py:7:1: F401 'time' imported but unused
```

Reply:
Ah didn't see those as the CI was blocked, ack I will address it